pkg/cmd/admin/node/evacuate.go

```diff
@@ -129,18 +155,18 @@ func (e *EvacuateOptions) RunEvacuate(node *kapi.Node) error {
 
 		printer.PrintObj(&pod, e.Options.Writer)
 
-		if foundrc || e.Force {
+		if isManaged || e.Force {
 			if err := e.Options.Kclient.Pods(pod.Namespace).Delete(pod.Name, deleteOptions); err != nil {
 				glog.Errorf("Unable to delete a pod: %+v, error: %v", pod, err)
 				errList = append(errList, err)
 				continue
 			}
 		} else { // Pods without replication controller and no --force option
-			numPodsWithNoRC++
+			numUnmanagedPods++
 		}
 	}
-	if numPodsWithNoRC > 0 {
-		err := fmt.Errorf(`Unable to evacuate some pods because they are not backed by replication controller.
+	if numUnmanagedPods > 0 {
+		err := fmt.Errorf(`Unable to evacuate some pods because they are not managed by replication controller or replica set or deaemon set.
```

Comment:
"because they are not managed by any type of controller (replication controller, replica set, daemon set, pet set)" or something similar